cmd/snap-gpio-helper/cmd_export.go

```diff
@@ -33,5 +38,38 @@ type cmdExportChardev struct {
 }
 
 func (c *cmdExportChardev) Execute(args []string) error {
-	return errors.New("not implemented")
+	chipLabels := strings.Split(c.Args.ChipLabels, ",")
```

Comment:
Since we are running other programs I think this could use `signal.NotifyContext` and pass that context to all the functions that need it.

Reply:
Good point, Thanks!